samples/peripheral/802154_phy_test/src/rf_proc.c

```diff
@@ -96,8 +104,7 @@ static void rf_ed_detected_fn(struct k_work *work)
 }
 
 #if CONFIG_PTT_ANTENNA_DIVERSITY
-#warning "Antenna diversity enabled but not yet supported in NCS"
-/* TODO: Implement when antenna diversity is supported in NCS */
+#pragma message "Antenna diversity is an experimental feature"
```

Comment:
It looks like a GCC extension. Is it safe to use it with other toolchains?

Reply:
With the note in Kconfig that this feature is experimental, I'm not sure if it's necessary to add compile-time warning. Maybe we can just remove this like?